Infrastructure/Models/Address.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Infrastructure.Models
+{
+    public class Address
+    {
+        public int? Id { get; set; }
```

Comment:
Should this be nullable? We will probably get into problems if someone creates entry without ID and then we make it "required" ?